### PR TITLE
Fix/remove auto save

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: VotingApplication/VotingApplication.Web/Views/Routes/Manage.cshtml [hunk 1]
```diff
@@ -9,7 +9,18 @@
...
-                <span class="icon-padded-left"><input class="manage-name-input" ng-model="poll.Name" required ng-model-options="{debounce: 1000}" ng-change="updatePoll()" /></span>
+                <form ng-submit="toggleNameEdit=false;updatePoll()">
+                    <span class="icon-padded-left">
+                        <span ng-if="!toggleNameEdit">
+                            {{poll.Name}}
+                            <span class="pull-right"><a class="clickable" ng-click="$parent.toggleNameEdit=true">Edit</a></span>
+                        </span>
+                        <span ng-if="toggleNameEdit">
+                            <input class="manage-name-input" ng-model="poll.Name" required />
+                            <span class="glyphicon glyphicon-ok pull-right clickable form-aligned-top" ng-click="$parent.toggleNameEdit=false;$parent.updatePoll()"></span>
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Views/Routes/Manage.cshtml]: We need a "Cancel" button to go along with this

Comment (stevenhillcox) [inline VotingApplication/VotingApplication.Web/Views/Routes/Manage.cshtml]: Cancelling is the same as typing the original name in. Alternatively, Ctrl + Z works just as well.

Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Views/Routes/Manage.cshtml]: Ctrl+Z is a tech-savvy shortcut, so we shouldn't rely on it. We also shouldn't rely on people remembering what the original name was.

**Comment by tpkelly** [comment]
We also need to rename "Back" to "Cancel" to be less ambiguous

**Pull request merged**